src/renderers/shaders/ShaderChunk/envmap_physical_pars_fragment.glsl.js

```diff
@@ -32,7 +32,7 @@ export default /* glsl */`
 
 			vec4 envMapColor = textureCubeUV( envMap, reflectVec, roughness );
 
-			return envMapColor.rgb * envMapIntensity;
+			return envMapColor.rgb;
```

Comment:
With this PR, the methods `getIBLRadiance()`, `getIBLIrradiance()`, and `getIBLAnisotropyRadiance()` are no longer returning the units their names imply.

Reply:
Ok, got it (was not aware that the multiplication is also a unit conversion). I could revert everything there and keep the scaling / unit conversion inside these functions and thus revert all changes in `lights_fragment_maps.glsl.js`. Instead initialize `envMapIntensityFactor` in `lights_fragment_maps.glsl.js` as `1.0`? This way it would be much less „invasive“.

Reply:
I pushed a less invasive version that requires no changes in `lights_fragment_maps.glsl.js`